meeshkan/serve/mock/faker/stateful_faker.py

```diff
@@ -0,0 +1,107 @@
+import typing
+from dataclasses import dataclass
+
+from http_types import Request
+from openapi_typed_2 import Any, Operation
+
+from meeshkan.serve.mock.faker.stateless_faker import FakerData, StatelessFaker
+from meeshkan.serve.mock.storage.entity import Entity
+from meeshkan.serve.utils.opanapi_ext import ApiOperation, get_x
+
+
+@dataclass(frozen=True)
+class StatefulFakerData(FakerData):
+    """
+    Common data passed through recursion steps in the StatefulFaker object.
+    """
+
+    entity: typing.Optional[Entity]
+    """
+    An entity related to a faked endpoint
```

Comment:
Should this docstring belong to the `entity` field above?

Reply:
Yes, docstrings are written below declarations as far as I get it.

Reply:
Ah ok, just not used to seeing it like this! I guess it's more a matter of convention then where to put the docstrings of attributes as they're not part of the class docstring anyway. I guess the "proper" place would in the main class docstring but not sure that's helpful at all.